src/Components/samples/ComponentsApp.App/ComponentsApp.App.csproj

```diff
@@ -1,20 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
-    <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <OutputType>Exe</OutputType>
-    </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputType>Library</OutputType>
```

Comment:
Is this needed?

Reply:
Great point. It was needed only because the SDK was set to `Microsoft.NET.Sdk.Web`, which it shouldn't have been. I've changed the SDK to `Microsoft.NET.Sdk.Razor` and now we no longer need to override `OutputType`.